v1/fs/index.ts

```diff
@@ -0,0 +1,38 @@
+import path from 'path'
+import fs from 'fs'
+import tar from 'tar-fs'
+import gunzip from 'gunzip-maybe'
+
+export class SiteFileSystem {
+  path: string
+  constructor (rootPath: string) {
+    this.path = path.join(rootPath, 'sites')
+  }
+
+  async clear (siteId: string): Promise<void> {
+    const sitePath = this.getPath(siteId)
+    fs.rmSync(sitePath, { recursive: true, force: true })
```

Comment:
This should probably be async so we can clear a bunch of stuff at once like the config DB in addition to the files.